services/user.js

```diff
@@ -1,8 +1,9 @@
-const {getCurrentTimeYYYYMMDDSS} = require("../utility/time-utility");
+
 const {v4} = require("uuid")
 const {USER} = require("../constants/user-constants");
 const {ERROR} = require("../constants/error-constants");
 const {UpdateProfileEvent} = require("../domain/log-events");
+const {getCurrentTimeYYYYMMDDSS, toISO} = require("../utility/time-utility");
```

Comment:
getCurrentTimeYYYYMMDDSS function should be renamed to something like getCurrentDateTime.

Reply:
updated!